_episodes/08-development-setup.md

```diff
@@ -25,9 +25,9 @@ ESMValTool is an open-source project in ESMValGroup. We can contribute to its de
 - a new or updated recipe script, see lesson on
 [Writing your own recipe]({{ page.root }}{% link _episodes/05-preprocessor.md %})
 - a new or updated diagnostics script, see lesson on
-[Writing your own diagnostic script]
+[Writing your own diagnostic script]({{ page.root }}{% link _episodes/10-diagnostics.md %})
 - a new or updated cmorizer script, see lesson on
-[CMORization: Using observational datasets]
+[CMORization: Using observational datasets]({{ page.root }}{% link _episodes/09-cmorization.md %})
 - helping  with reviewing process of pull requests, see ESMValTool documentation on
 [Review of pull requests](https://docs.esmvaltool.org/en/latest/community/review.html)
 
```

Comment:
In this lesson, we first show how to set up a development installation of ESMValTool so you can make changes or additions.

Reply:
We then explain how you can contribute these changes to the community.

Reply:
The paragraph is now revised as you suggested.